pegasus/sites.v3/hourofcode.com/public/index.haml

```diff
@@ -96,10 +96,6 @@ social:
                 =hoc_s(:front_join_us_n_students_served).gsub("#", format_integer_with_commas(fetch_hoc_metrics['started']).to_s)
             %div.ctabuttongroup
               %div#tryittext.ctatext
-                %a#signupbutton.ctabuttonatag{onclick: "adjustScroll('join');", style: "text-decoration: none;"}
```

Comment:
if we're hiding in the below file, should we also hide here to be consistent, rather than delete?